package.json

```diff
@@ -11,7 +11,8 @@
     "build": "gatsby build",
     "serve": "gatsby serve",
     "serve:fresh": "yarn clean; yarn build; yarn serve",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "fmt:docs": "prettier --config .prettierrc.js --write \"{!**/*.{js,jsx,ts,tsx,json},**/*.{md,mdx}}\"",
+    "fmt:site": "prettier --config .prettierrc.js --write \"{**/*.{js,jsx,ts,tsx,json},!**/*.{md,mdx}}\"",
```

Comment:
New format commands. Normally we will want `fmt:docs` for the devhub team and `fmt:site` for the webdev team
**IMHO these should be run this as a CI item**

I added a style guide note on this for contributors as well